avocado/utils/process.py

```diff
@@ -449,7 +449,7 @@ def _fd_drainer(self, input_pipe, ignore_bg_processes=False):
                 # Don't read unless there are new data available:
                 if not select.select([fileno], [], [], 1)[0]:
                     continue
-            tmp = os.read(fileno, 1024)
+            tmp = os.read(fileno, 4096)
```

Comment:
Well, 1kb is not really much, 4kb is not a big change. How about changing it to 10k right away?